test/e2e/ExpectModelUtils.ts

```diff
@@ -86,15 +119,121 @@ export interface ExpectedStartEventModelElement extends ExpectedEventModelElemen
 
 export const bpmnVisualization = new BpmnVisualization(null);
 
-function toBeCell(this: MatcherContext, received: string): CustomMatcherResult {
-  const cell = bpmnVisualization.graph.model.getCell(received);
-  const pass = cell ? true : false;
+// -------------------------- Convert objects to compare ---------------------------
+function buildExpectedStateStyle(expectedModel: ExpectedEdgeModelElement): ExpectedStateStyle {
+  const font = expectedModel.font;
+  const fontStyle = font && (font.isBold || font.isItalic || font.isStrikeThrough || font.isUnderline) ? getFontStyleValue(font) : undefined;
+  return {
+    verticalAlign: expectedModel.verticalAlign ? expectedModel.verticalAlign : 'top',
+    align: 'center',
+    strokeWidth: 1.5,
+    strokeColor: 'Black',
+    fillColor: 'White',
+    rounded: 1,
+    fontFamily: font?.name ? font.name : 'Arial, Helvetica, sans-serif',
+    fontSize: font?.size ? font.size : 11,
+    fontColor: 'Black',
+    fontStyle: fontStyle,
+    startArrow: expectedModel.startArrow,
+    endArrow: expectedModel.endArrow,
+    endSize: 12,
```

Comment:
Before this change, we didn't test all style properties all the time  as it is already tested with the StyleConfigurator. I fear this will be a maintenance issue: we have the visual tests to ensure the rendering is ok. The actual style were are setting in the model is just implementation, we mainly care about the rendering result.
Anyway, we will see.